### DISCUSSION TITLE
What theme is using DG Docs web?

### DISCUSSION THREAD

**Discussion by David7ce** [oleeskild/obsidian-digital-garden#252] [Q&A] [answered]
I would like to use a similar theme to the [Documentation web of Obsidian Digital Garden](https://dg-docs.ole.dev/).
Grey background and bold letters yellow.
But I don't know the name of the theme.

And also what is the name of the default theme of Obsidian (purple and black) because the default theme of Digital Garden plugin is different, with title in orange and other colors.

I would like to see a preview of themes or at least how it looks. I looked for the theme names on Obsidian and see the preview, but not all of them correspond to the same name or they don't exist in Obsidian.

**Comment by shinanqu** [comment, marked as answer]
I think this will answer your question:
https://github.com/oleeskild/obsidian-digital-garden/discussions/232

**Reply by David7ce** [reply]
Ok, thanks. And which is the Obsidian theme (purple and black)?
